### PR TITLE
[9.0][ADD] medical_pathology_import: Add importer interface

### PR DESCRIPTION
**Pull request by lasley**
* Add module to provide an import interface for external medical pathology data

Depends:
- [x] #138 
- [x] #135

### CODE REVIEW

File: medical_pathology_import/wizards/medical_pathology_import.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+from openerp import api, fields, models
+
+
+class MedicalPathologyImport(models.TransientModel):
+    """ It provides a model for importing ICD-10-CM codes to Odoo. """
+
+    _name = 'medical.pathology.import'
+    _description = 'Medical Pathology Import'
+
+    zip_uri = fields.Char(
```
Comment (lasley) [inline medical_pathology_import/wizards/medical_pathology_import.py]: Move zip extraction method into this from icd10

File: medical_pathology_import/wizards/medical_pathology_import.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        help='URI to full Zip file.',
+    )
+    file_name = fields.Char(
+        help='Name of file (inside of zip) representing full tabular xml.',
+    )
```
Comment (lasley) [inline medical_pathology_import/wizards/medical_pathology_import.py]: Add user & pass to support SNOMED (UMLS is password protected)

**Review by susport — COMMENTED** [review]
Which is the format of the file to import of external Pathology. Cant you validate this?

**Comment by lasley** [comment]
The files are going to be different depending on the importer interface. Most are either CSV or XML files, and typically they're contained in ZIP files. You can see an example of an importer here - https://github.com/OCA/vertical-medical/pull/134/files#diff-35868cdb8c46f43d773151b6ec6c7f56

**Comment by susport** [comment]
Thank you @lasley . 👍

**Comment by lasley** [comment]
Rebased now that dependency is merged; this is ready for review.

File: medical_pathology_import/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+===============================
+Odoo Medical Pathology Importer
+===============================
+
+This module adds a wizard and interface for the import of external Pathology
+data.
+
+Usage
+=====
+
```
Comment (lasley) [inline medical_pathology_import/README.rst:14]: Improve this, add where the wizard is & how to use.

Also maybe add a note of how to implement when creating adapters

File: medical_pathology_import/views/medical_menu.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <menuitem id="medical_pathology_import_menu"
```
Comment (lasley) [inline medical_pathology_import/views/medical_menu.xml:7]: This should be in Settings=>Technical settings

File: medical_pathology_import/README.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
+
+Known issues / Roadmap
+======================
+
+* Add translation support for imported data (https://github.com/OCA/vertical-medical/issues/136)
+
```
Comment (lasley) [inline medical_pathology_import/README.rst:23]: Add an action that takes user to the import wizard after install

**Comment by lasley** [comment]
Some of this needs to be split into a `medical_base_import` in order to support SNOMED and RxNorm data from UMLS, which includes medical terminologies spanning from pathologies to medicaments.

**Comment by lasley** [comment]
Closing - we're going to resubmit under v10

**Pull request closed without merging**